04-behavr.Rmd

```diff
@@ -129,7 +130,7 @@ summary(dt, detailed = TRUE)
 Playing with variables is just like in `data.table`.
 Read the [official data.table tutorial](https://cran.r-project.org/web/packages/data.table/vignettes/datatable-intro.html) for more functionalities.
 
-For instance, we can add a **new variable**, `very_active`, that is `TRUE` if and only if
+For instance, we can add a **new variable**, `very_active`, that is `TRUE` if, and only if,
```

Comment:
I will revert this as if and only if is a mathematical phrase, I don't think we need comas: https://english.stackexchange.com/questions/18059/usage-of-if-and-only-if-as-a-nontechnical-term